nautobot-app/{{ cookiecutter.project_slug }}/tasks.py

```diff
@@ -201,6 +203,23 @@ def build(context, force_rm=False, cache=True):
     docker_compose(context, command)
 
 
+def _ensure_creds_env_file(context):
+    """Ensure that the development/creds.env file exists."""
+    if not os.path.exists(
+        os.path.join(context.{{ cookiecutter.app_name }}.compose_dir, "creds.env")
+    ):
+        # Warn the user that the creds.env file does not exist and that we are copying the example file to it
+        print(
+            "⚠️⚠️ The creds.env file does not exist, using the example file to create it. ⚠️⚠️"
```

Comment:
I don't know if it's just my terminal but the spacing between these emojis isn't working correctly. Take a look at the diff when I add a space in between them:

![image](https://github.com/user-attachments/assets/c2256c42-cf31-44f4-877a-81621b58c899)

Reply:
The emojis are overlapped on my terminal too when I look at the diff.